demo/vl-map-layer.html

```diff
@@ -213,7 +213,7 @@ <h2>Kaart met kaartlaag met features en auto extent</h2>
                                     "coordinates": [151055.0,201908.0]
                                 }
                             }]
-                        }' auto-extent>
+                        }' auto-extent auto-extent-max-zoom="3">
```

Comment:
moesten niet alle attributen met data-vl beginnen @coemans? of wordt dat nog eens een apart rondje webcomponenten?

Reply:
Ik dacht vroeger gehoord te hebben dat de componenten waar het al is zo te laten (tot we een keer het in een bulk change veranderen).

Reply:
Ticket staat (zeer binnenkort) op de planning, dus inderdaad best voorlopig consistent blijven.